src/coins.json

```diff
@@ -1061,3 +1061,12 @@
     "symbol_with_bridge": "Woof"
   }
 ]
+    "chainId": 1,
```

Comment:
not json

Reply:
fixed in latest commit. sorry about that.